tests/behat/features/deputy/01-registration-steps/04-add-client-and-report.feature

```diff
@@ -172,3 +172,14 @@ Feature: deputy / user / add client and report
     Given I click on "reports, report-2016-open"
     Then the URL should match "report/\d+/overview"
     And I save the page as "deputy-report-overview"
+
+  @deputy
+  Scenario: report-overview links
+    Given I am logged in as "behat-user@publicguardian.gsi.gov.uk" with password "Abcd1234"
+    #Lay deputy links
+    Then I should see the "user-account" link
+    Then I should see the "reports" link
+    Then I should see the "logout" link
+    #PA links
+    Then I should not see the "pa-dashboard" link
```

Comment:
I'd reword a bit the "Then" prefixes
e.g.

Then I should see the "user-account" link
And I should see the "reports" link
And I should see the "logout" link
But I should not see the "pa-dashboard" link